util/job_manager.rb

```diff
@@ -48,6 +48,7 @@ def perform
       OptimizerWrapper::REDIS.set(Resque::Plugins::Status::Hash.status_key(self.uuid), Resque::Plugins::Status::Hash.encode(value))
 
       ask_restitution_csv = services_vrps.any?{ |s_v| s_v[:vrp].restitution_csv }
+      ask_restitution_geojson = services_vrps.collect{ |s_v| s_v[:vrp].restitution_geometry }.flatten.uniq
```

Comment:
flat_map

Reply:
in fact, `any?` should work as the way it works for `ask_restitution_csv`, no?

Reply:
s_v[:vrp].restitution_csv is a boolean while s_v[:vrp].restitution_geometry can be an empty array and, in this case, we do not want to call function.

Well in fact that was initial idea. Now we always want to provide, at least, points. So in fact we do not even need this anymore.